crates/common/src/raindex_client/vaults.rs

```diff
@@ -319,17 +330,21 @@ impl RaindexVault {
     /// const calldata = result.value;
     /// // Do something with the calldata
     /// ```
-    #[wasm_export(js_name = "getWithdrawCalldata", unchecked_return_type = "Hex")]
+    #[wasm_export(
+        js_name = "getWithdrawCalldata",
+        return_description = "Encoded transaction calldata as hex string",
+        unchecked_return_type = "Hex"
+    )]
     pub async fn get_withdraw_calldata(
         &self,
-        withdraw_amount: String,
+        #[wasm_export(param_description = "Amount to withdraw")] amount: String,
```

Comment:
same here - it doesn't say how this amount should be provided but the user facing format makes sense

Reply:
same comment as above. although we can be more specific with the current format. i can update the docs to be more descriptive as above comment